development/app/Makefile

```diff
@@ -1,13 +1,22 @@
-all: build
```

Comment:
`build` is not present as target, so renamed this to `docker-build-all` with all the `docker-build-*` targets

Reply:
this makes sense. thanks for the improvements